examples/advanced/bionemo/downstream/tap/jobs/central_tap_esm1nv/app1/custom/downstream_flip_tap.yaml

```diff
@@ -1,11 +1,32 @@
 name: esm1nv_flip
```

Comment:
if all the "downstream_flip_tap.yaml" is the same, we could in next PR use job api or job template to generate the job.
that way we only need to put one copy on GitHub

Reply:
yes this is a suggestion I already made to holger, but for the sake of time we went ahead to just make it functional

Reply:
Good point. Let me file an issue.